.gitignore

```diff
@@ -357,3 +357,11 @@ tool/
 
 # ApiCompat suppression files
 suppress.*.xml
+
+# Connetor analysis reports
+Analysis.txt
+Report.json
+Report_*.json
+
+# global.json
+global.json
```

Comment:
what file is this?

Reply:
[https://learn.microsoft.com/en-us/dotnet/core/tools/global-json](https://learn.microsoft.com/en-us/dotnet/core/tools/global-json)

It's set during build pipeline but I don't want it to be checked-in if I define it locally